app/models/solidus_stripe/slug_entry.rb

```diff
@@ -14,24 +14,7 @@ module SolidusStripe
   #
   # We use a slug instead of the payment method ID to be resilient to
   # database changes and to avoid guessing about valid endpoint URLs.
-  class WebhookEndpoint < ::Spree::Base
-    belongs_to :payment_method,
-      class_name: 'SolidusStripe::PaymentMethod'
-
-    # @api private
-    def self.generate_slug
-      SecureRandom.hex(16).then do |slug|
-        exists?(slug: slug) ? generate_slug : slug
-      end
-    end
-
-    # Finds the payment method associated with the given slug.
-    #
-    # @param slug [String]
-    # @raise [ActiveRecord::RecordNotFound] if no payment method is found
-    # @return [SolidusStripe::PaymentMethod]
-    def self.payment_method(slug)
-      find_by!(slug: slug).payment_method
-    end
+  class SlugEntry < ::Spree::Base
+    belongs_to :payment_method, class_name: 'SolidusStripe::PaymentMethod'
```

Comment:
Maybe I'm missing something, but I do not generally understand why we have a model + table SlugEntry/WebhookEndpoint and not a simple `slug` column in `spree_payment_methods`?

Reply:
@rainerdema it's only because the payment methods table uses STI and we don't want to pollute it for every other payment method. Alternatively we could have used preferences but those can't be used in queries as they're serialized as YAML.

Reply:
Makes total sense 🙌
> Alternatively we could have used preferences but those can't be used in queries as they're serialized as YAML.

Thinking about preferences, I think it can still be a good compromise given their purpose 🤔 
However, these changes fully represent the purpose of the related issue, and maybe we can discuss it in a separate PR.

Thanks @elia 🙏